CHANGELOG.md

```diff
@@ -85,6 +85,7 @@
    * FIXED: Multicue enter roundabout [#2556](https://github.com/valhalla/valhalla/pull/2556)
 
 * **Enhancement**
+   * ADDED: Dependency cache for Azure [#2567](https://github.com/valhalla/valhalla/pull/2567)
```

Comment:
we've been putting these as newest are last. we should probably document that, sorry for the confusion

Reply:
Oops, did that for all my PRs.. Should I clean that up?

Reply:
if you want to or if you'd rather wait for the next one no problem. not a huge deal, just letting you know for future reference

Reply:
done